### DISCUSSION TITLE
make last 5 columns in table sticky

### DISCUSSION THREAD

**Discussion by manimovassagh** [quasarframework/quasar#14177] [General - Components / Directives / etc] [answered]
Hi communtiy,
I am using in my app quasar table and i want to make the last 5 columns inside the table sticky.
as mentioned in documents i used the css selectors.
i can grab the last 5 columns and change the background color but sticky only work on last one.
Codepen regenration:
https://codepen.io/manimovassagh/pen/zYWJPqq
I will be very thankfull if somebody nows how can i do it .
Thanks

**Comment by metalsadman** [comment, marked as answer]
https://codepen.io/metalsadman/pen/OJvoOQd

**Comment by manimovassagh** [comment]
Thanks a lot it really helped.
only one problem :at the left side i have dynamic number of columns .
is there any way that i count from end 
for example instead of 
 th:nth-child(6),
  td:nth-child(6) {
    right: 190px;
  }
use 
 th:nth-child(-n+5),
  td:nth-child(-n+5) {
    right: 190px;
  }
I mean count and select from the end of table.
Thanks

**Comment by manimovassagh** [comment]
Hi,
I have found it
https://codepen.io/metalsadman/pen/vYRzWpr
Really Thanks for your help
👍👍👍

**Reply by metalsadman** [reply]
i updated the codepen and used a for loop, should make the css shorter.

**Reply by manimovassagh** [reply]
Thanks really helped me out 😊😊
